chia/cmds/chia.py

```diff
@@ -91,7 +92,8 @@ def cli(
         except Exception as e:
             print(f"Failed to read passphrase: {e}")
 
-    check_ssl(Path(root_path))
+    if config_path_for_filename(Path(root_path), "config.yaml").exists():
```

Comment:
Without this check, `chia version` (or similar commands) will fail on a fresh install (no `~/.chia dir`)